ui/src/kapacitor/components/RuleGraph.js

```diff
@@ -53,6 +53,21 @@ export const RuleGraph = React.createClass({
     )
   },
 
+  _getFillColor(operator) {
+    const backgroundColor = 'rgba(41, 41, 51, 1)'
+    const highlightColor = 'rgba(78, 216, 160, 0.3)'
+
+    if (operator === 'outside range') {
```

Comment:
i think this and the `not equal to` below should be constants.  Here are the constants: 
https://github.com/influxdata/chronograf/blob/870dbc72d1a8b784eaacad5eeea79fc54968b656/kapacitor/operators.go#L13